modules/googleplay/src/test/scala/cards/nine/googleplay/service/free/interpreter/cache/InterpreterSpec.scala

```diff
@@ -53,11 +54,11 @@ class InterpreterSpec
 
     import scala.concurrent.ExecutionContext.Implicits.global
 
-    val interpreter = new CacheInterpreter()
+    val cc = new CacheInterpreter()
 
-    def eval[A](op: Ops[A]) = interpreter(op)(redisClient).unsafePerformSync
+    def eval[A](op: RedisOps[A]) = op(redisClient).unsafePerformSync
```

Comment:
Can we add the return type `A` here for `eval` and for `evalWithDelay`, it might not be obvious for everyone that `RedisOps` is a type alias for `Kleisli[Task, ScredisClient, ?]`.